src/interface/src/app/backend-constants.ts

```diff
@@ -0,0 +1,3 @@
+export class BackendConstants {
```

Comment:
Instead of making a class, this can be
export const BackendConstants = { 
  HOST: 'http://localhost:8000',
} as const;

Compared with creating a class, the advantage of const is that it makes variables immutable for both compile time and run time.

Reply:
done.